### ISSUE TITLE
unauthorized

### ISSUE THREAD

**Issue by fehim-proses** [IzakMarais/reporter#317]
error fetching dashboard BX3Gpdbnz: error obtaining dashboard from http://localhost:3000/api/dashboards/uid/BX3Gpdbnz. Got Status 401 Unauthorized, message: {
  "message": "Unauthorized"
}

I got this message

**Comment by khaitun1998** [comment]
You should create api token in grafana, then pass it into url as query string

Ex: 

`
http://localhost:3000/api/dashboards/uid/BX3Gpdbnz?apitoken={grafana_token}
`

**Comment by ObserveLite** [comment]
@khaitun1998  yes but the Grafana Open Source doesn't give the API token for Users
the API token is only available in Enterprise and Grafana cloud.

**Comment by tommeeeee** [comment]
> @khaitun1998 yes but the Grafana Open Source doesn't give the API token for Users the API token is only available in Enterprise and Grafana cloud.

@fehim-proses 

You can generate you api key 

Go to Administration -> Users and access -> Service and accounts -> Add service and account -> Select Roles -> Add Service account token (API Key)
